### PR TITLE
Add topologySpreadConstraint for node-wizard 

### PR DESCRIPTION
**Pull request by alperencelik**
Hi,

First of all thanks for the great project! I really appreciate the idea of the project. 

  I was doing some tests about the node-wizard's helm chart and found out that it doesn't have any specialized scheduler rule. I thought that since it's an app to check node health the application should distribute as much as possible for different nodes so even though multiple nodes go down, the application can have much more chance to live since its replicas are distributed.

### CODE REVIEW

**Comment by alperencelik** [comment]
For adding POD_NAMESPACE field as environment variable please see the PR https://github.com/cnwizards/node-wizard/pull/5

File: charts/node-wizard/values.yaml
Comment (pehlicd) [inline charts/node-wizard/values.yaml:1]: Could you fix the lint?

**Comment by pehlicd** [comment]
Hey @alperencelik, thanks for the PR you are completely true about scheduler rule thing. Everything seems fine except for the lint issue. Could you fix it?

File: charts/node-wizard/values.yaml
Comment (pehlicd) [reply charts/node-wizard/values.yaml:1]: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.trailing_spaces related doc.

**Review by pehlicd — APPROVED** [review]
LGTM!

**Pull request merged**